auth/api/iam/api.go

```diff
@@ -760,9 +763,10 @@ func (r Wrapper) RequestServiceAccessToken(ctx context.Context, request RequestS
 	if tokenResult.ExpiresIn != nil {
 		ttl = time.Second * time.Duration(*tokenResult.ExpiresIn)
 	}
+	expiresAt := time.Now().Add(ttl)
```

Comment:
can't we just add expires_at to the API spec?

Reply:
yeah why not.